modules/ocm-nw-networkpolicy-view.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * networking/network_policy/viewing-network-policy.adoc
+// * post_installation_configuration/network-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="ocm-nw-networkpolicy-view_{context}"]
+= Viewing {name} policies using {cluster-manager}
+
+You can view your network policy's configuration details in {cluster-manager-first}.
```

Comment:
You could update "your network policy's configuration details" to the following, to avoid using a possessive:

> the configuration details of your network policy

My understanding is that this facilitates localisation.